content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst

```diff
@@ -1,77 +1,107 @@
 
 .. image:: using_inventory_valuation/use_inventory_valuation_03.png
     :align: center
 
-I can also easily access the accounting entry that has been generated
-(in case of automated inventory valuation).
+In Odoo, automatic inventory valuation records are also recorded in the :guilabel:`Accounting` app.
+To access these accounting entries,navigate to :menuselection:`Accounting --> Journal Entries`, and
+look for `Inventory Valuation` entries with the `STJ` prefix in the :guilabel:`Journal` and
+:guilabel:`Number` columns, respectively.
+
+For this example, search the :guilabel:`Reference` column for the :guilabel:`Receipt` `WH/IN/00001`
+to select the record for the inventory valuation of 10 tables. Below is the accounting entry.
 
 .. image:: using_inventory_valuation/use_inventory_valuation_04.png
     :align: center
 
 Deliver a product
 -----------------
 
-In the same logic, when a table will be delivered, the stock valuation
-will be impacted and you will have access to a similar information.
+In the same logic, when a table is shipped to a customer and leaves the warehouse, the stock
+valuation decreases. The :guilabel:`Valuation` smart button on the :abbr:`DO(Delivery Order)` displays the stock valuation record.
 
 .. image:: using_inventory_valuation/use_inventory_valuation_05.png
     :align: center
 
 The inventory valuation report
 ------------------------------
 
-The summary of this is accessible via the inventory valuation report
-(accessible from :menuselection:`Inventory --> Reporting --> Inventory Valuation`). It gives
-you, product per product, the value of your stock. By clicking on the
-button *Inventory At Date*, you can have the same information for a
-past date.
+To view the current value of all products in the warehouse, go to :menuselection:`Reporting -->
+Inventory Valuation` in the :guilabel:`Inventory` app. The records are organized by product,
+and selecting the drop-down displays detailed records with the date, quantity, and value of the
+shipment.
 
 .. image:: using_inventory_valuation/use_inventory_valuation_06.png
     :align: center
+
+.. note::
+    :guilabel:`Inventory At Date` shows the inventory valuation of products available during a past
+    date.
+
+Update product unit price
+~~~~~~~~~~~~~~~~~~~~~~~~~
+For any company, the lead time, risk of supply chain failures, and other factors contribute to
+invisible costs. Although Odoo attempts to accurately represent the stock value, *manual valuation*
+serves as an additional tool to update the unit price of products.
+
+This operation is intended for products that can be purchased and received for a cost greater than
+0, or have product categories set with :guilabel:`Costing Method` set as either `Average Cost
+(AVCO)` or `First In First Out (FIFO)`.
+
+.. image:: using_inventory_valuation/use_inventory_valuation_07.png
+    :align: center
+
+The Product Revaluation form updates the calculation of inventory valuation by increasing or
```

Comment:
Lets :guilabel:Product Revaluation because we are referring to it directly in this sentence

Reply:
Not sure this applies...the logic is correct but it's a gray area since it's not a heading, button, filter, label, etc.
I'll ask JCS